tests/test_oom_logger.py

```diff
@@ -344,14 +345,21 @@ def test_send_sfx_event(mock_get_instance_config):
         assert mock_meteorite.create_counter.return_value.count.call_count == 1
 
 
+@patch("paasta_tools.oom_logger.argparse", autospec=True)
+def test_parse_args(mock_argparse):
+    assert parse_args() == mock_argparse.ArgumentParser.return_value.parse_args()
```

Comment:
i think it's fine to leave this untested since we're not really checking for much here :)